docs/docs/07-Development/01-setup.md

```diff
@@ -60,4 +60,13 @@ In dev mode, opening and closing the plugin menu should reload the code.
 
 ## Docker Dev Env
 
-If the manual setup is too much hassle for you. You can use a docker based dev environment by running `docker compose -f docker/docker-compose.dev.yml up` in the root of the repo. This setup wasn't super reliable for me though.
+If the manual setup is too much hassle for you, you can use a docker based dev environment instead by following the instructions:
+
+- Clone the repo and navigate to the root of the repo.
+- Run `cp /docker/.env.sample /docker/.env`
+- Make the necessary changes to the .env file (setting the DOCKER_DATA_DIR, NEXTAUTH_SECRET, and MEILI_MASTER_KEY)
```

Comment:
You don't really need to set nextauth_secret or even meili key during development. So I wouldn't worry too much about them

Reply:
Wouldn't it be a good idea to set it up anyways, just to make sure that any development does not break functionality offered by nextauth or meilisearch?

Reply:
I am too prefer having nextauth and meilisearch being ready while developing

Reply:
Just to be clear. They will be ready and usable. The requirements of the variables being set are only when deployed to prod.

Reply:
oh gotcha because you want that stuff seeded with a random value